lib/crystalball/map_generator/helpers/method_replacer.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Crystalball
+  class MapGenerator
+    module Helpers
+      # Helper module to patch and restore methods
+      module MethodReplacer
+        # Add prefixes 'cb_original_' to original methods to backup them.
+        # Replaces original methods with 'cb_patched_' methods
+        # @param Symbol method_names - list of method names to patch
```

Comment:
I think those lines should be:
```ruby
# @overload replace_with_patch(method_name, ...)
#   Add prefixes 'cb_original_' to original methods to backup them.
#   Replaces original methods with 'cb_patched_' methods
#   @param [Symbol] method_name - name of method to patch
#   @param [Symbol] ... - other names of methods to patch
```
See [this](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#overload).